reference/orders.v2.oas2.yml

```diff
@@ -1058,6 +1079,363 @@ paths:
+                        base_cost: '4.0000'
+                        cost_ex_tax: '4.0000'
+                        cost_inc_tax: '12.0000'
+                        cost_tax: '8.0000'
+                        tax_class_id: 0
+        '404':
+          $ref: '#/components/responses/404_Resp'
+      operationId: getOrderFees
+      description: 'Get all fees for an order. '
+      parameters:
+        - $ref: '#/components/parameters/order_id_path'
+  '/orders/{order_id}?include=fees':
+    parameters:
+      - name: include
+        in: query
+        description: |-
+          * `fees` - include the response returned from the request to the `/orders/{order_id}/fees`.
+        style: form
+        explode: false
+        schema:
+          type: array
+          items:
+            type: string
+            enum:
+              - fees
+    get:
+      summary: Get Fees
+      tags:
+        - Order Fees
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/orderFees'
+              examples:
+                response:
+                  value:
+                    fees:
+                      - id: 1
+                        type: custom_fee
+                        display_name_customer: A Fee
+                        display_name_merchant: A Fee
+                        source: v2
+                        base_cost: '12.3000'
+                        cost_exc_tax: '12.3000'
+                        cost_inc_tax: '12.30000'
+                        cost_tax: '0.0000'
+                        tax_class_id: 22
+                      - id: 2
+                        type: custom_fee
+                        display_name_customer: B Fee
+                        display_name_merchant: B Fee
+                        source: v2
+                        base_cost: '4.0000'
+                        cost_ex_tax: '4.0000'
+                        cost_inc_tax: '12.0000'
+                        cost_tax: '8.0000'
+                        tax_class_id: 0
+        '404':
+          $ref: '#/components/responses/404_Resp'
+      operationId: getOrderFees
+      description: 'Get all fees for an order. '
+      parameters:
+        - $ref: '#/components/parameters/order_id_path'
+    put:
+      description: |-
+        To update the fees for an order, include the fee `id` in the request body. The body should include all relevant fields, with either the updated values or the existing values you wish to retain. Fees not included in the payload will be deleted. Fees provided in the request will be updated, and fees without an ID will be created as new.
+      summary: Update fees for an order
+      tags:
+        - Order Fees
+      parameters:
+        - $ref: '#/components/parameters/ContentType'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/orderFees'
```

Comment:
probably not all the fields in this object are supported in PUT/POST request

Reply:
Added separate schema for all